test/unit/spec/EnrollChoices_spec.js

```diff
@@ -736,6 +736,16 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
               expect(test.form.factorCardinalityText(factorName)).toBe('(3 set up)');
             });
         });
+        itp('checks correct order after enroll', function () {
+          return setupMultipleFactorEnrollments(
+            factorName, 'ACTIVE', 'OPTIONAL',{'enrolled': 1, 'minimum': 1, 'maximum': 3}
+          ).then(function (test) {
+            var factorRows = test.form.optionalFactorList();
+            var factorIndex = factorRows.find(`[data-se="${factorName}"]`).index();
+            var expectedIndex = factorName === 'U2F' ? 1 : 0;
```

Comment:
This may not work once more factors support cardinality. Would it be possible to use the sort order in FactorUtil to get the expectedIndex? https://github.com/okta/okta-signin-widget/blob/master/src/util/FactorUtil.js#L29

Reply:
Thanks for the note! Now I use FactorUtil, although I use a condition inside it anyway.